src/components/workspace/settings/UsedVolume.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <div class="workspace-volume settings-window-page">
+    <div class="settings-window-page__title">
+      {{ $t('workspaces.settings.volume.title') }}
+    </div>
+    <div class="workspace-volume__text">{{ $t('workspaces.settings.volume.description_1') }}</div>
+    <label class="label">{{ $t('workspaces.settings.volume.title') }}</label>
+    <div class="events-limit-indicator__events">
+      {{ eventsCount }} /
+      {{ plan.eventsLimit }}
+      {{ $tc("billing.volumeEvents", eventsCount) }}
+    </div>
+    <Progress
+      :max="plan.eventsLimit || 0"
+      :current="eventsCount"
+      :color="
+        eventsCount / (plan.eventsLimit || eventsCount) >= 0.9
+        ? 'var(--color-indicator-critical)'
+        : 'rgba(219, 230, 255, 0.6)'"
+      class="events-limit-indicator__volume-progress workspace-volume__volume-progress"
+    />
+    <div class="workspace-volume__text">{{ $t('workspaces.settings.volume.description_2') }}</div>
+    <a href="mailto:team@hawk.so">
+      <div class="button workspace-volume__button">{{ $t('workspaces.settings.volume.emailButton') }}</div>
```

Comment:
redundant `div`, you can add `button workspace-volume__button` classes to the `a` tag